teamB/simchaewon/routes/postRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const postController=require("../controllers/postController");
+
+const router=express.Router();
+router.post("/upload", postController.upload);
+router.get("/readall", postController.readall);
+router.patch("/update", postController.update);
+router.delete("/delete", postController.deletePost);
```

Comment:
코드 가독성을 위해 해당 부분은 한 줄을 띄어서 작성해 주세요!
```suggestion
const router=express.Router();

router.post("/upload", postController.upload);
router.get("/readall", postController.readall);
router.patch("/update", postController.update);
router.delete("/delete", postController.deletePost);
```